apps/elixir_chatbot_core/lib/chatbot.ex

```diff
@@ -30,7 +30,8 @@ defmodule ChatWeb.BotFacade do
     prompt =
       "<|USER|>#{fragments_text}\nIn the Elixir programming language, #{question}?<|ASSISTANT|>"
 
-    response = ElixirChatbotCore.GenerationModel.GenerationModel.generate(model, prompt, %{})
+    # response = ElixirChatbotCore.GenerationModel.GenerationModel.generate(model, prompt, %{})
+    response = "abc"
```

Comment:
Wait, are we sure we want to push this one to main?

Reply:
No, thanks for finding it, reverted